src/components/molecules/Slider/index.js

```diff
@@ -1,67 +1,79 @@
 import React, { useState } from 'react';
 import Button from '../../atoms/Button';
-import img1 from '../../../images/carousel_img_1.png';
-import img2 from '../../../images/carousel_img_2.png';
-import img3 from '../../../images/carousel_img_3.png';
+import CarouselText from '../CarouselText';
 import './style.scss';
 
-const Slider = () => {
-  const img = [
-    <img key={img1} src={img1} alt="img1" />,
-    <img key={img2} src={img2} alt="img2" />,
-    <img key={img3} src={img3} alt="img3" />,
+const Slider = ({ title, subtitle, images }) => {
+  const slides = [
+    <img key="1" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} />,
+    <img key="2" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} />,
+    <img key="3" src={images && require(`../../../images/${images[1].src}`).default} alt={images && images[1].alt} />,
+    <img key="4" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} />,
+    <img key="5" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} />,
   ];
+  const [transform, setTransform] = useState(-100);
+  const [transition, setTransition] = useState('transitionOn');
+  const [step, setStep] = useState(1);
 
-  const [activeIndex, setActiveIndex] = useState(0);
-  const [isDirectionRight, setRightDirection] = useState(true);
-
-  let prevImgIndex;
-  let nextImgIndex;
-
-  if (isDirectionRight) {
-    prevImgIndex = activeIndex ? activeIndex - 1 : img.length - 1;
-    nextImgIndex = activeIndex === img.length - 1 ? 0 : activeIndex + 1;
-  } else {
-    nextImgIndex = activeIndex ? activeIndex - 1 : img.length - 1;
-    prevImgIndex = activeIndex === img.length - 1 ? 0 : activeIndex + 1;
-  }
-
-  const dropDownClickRight = () => {
-    setRightDirection(true);
-    setActiveIndex((current) => {
-      const res = current === img.length - 1 ? 0 : current + 1;
-      return res;
-    });
+  const dropDownClickLeft = () => {
+    setStep(step - 1);
+    if (step < 1) {
+      setStep(slides.length - 1);
```

Comment:
I would prefer to have constant for `slides.length`

Reply:
Ok, thanks, i will change it.